toolkit/templates/forms/macros/forms.html

```diff
@@ -211,20 +215,20 @@
 {% macro number(question_content, data, errors) -%}
   {%
     with
-    name=question_content.id,
-    question=question_content.question,
-    optional=question_content.optional,
-    question_advice=question_content.question_advice,
-    hint=question_content.hint,
-    value=data[question_content.id],
-    error=errors.get(question_content.id)['message'],
-    hidden=question_content.hidden,
-    unit_in_full=question_content.unit_in_full,
-    unit_position=question_content.unit_position,
-    unit=question_content.unit,
-    smaller=True,
+    error=kwargs.get("error", errors.get(question_content.id)['message']),
+    hidden=kwargs.get("hidden", question_content.hidden),
+    hint=kwargs.get("hint", question_content.hint),
+    message=kwargs.message,
+    name=kwargs.get("name", question_content.id),
+    optional=kwargs.get("optional", question_content.optional),
+    question=kwargs.get("question", question_content.question),
+    question_advice=kwargs.get("question_advice", question_content.question_advice),
     question_number=kwargs.question_number,
-    message=kwargs.message
+    smaller=kwargs.get("smaller", True),
```

Comment:
This `smaller` is another one I'm not sure should be possible to override - this enforced `smaller` is pretty much what distinguishes this `number` macro from a regular textbox macro I think.

Reply:
I don't really feel it's my job here to enforce arbitrary restrictions. Any restrictions I put on I can see having to be overridden and it creating an extra piece of work to come here & change them.